backend/src/middleware/fileSizeValidator.middleware.js

```
@@ -0,0 +1,22 @@
+const { MAX_FILE_SIZE } = require('../utils/constants');
+
+const fileSizeValidator = (req, res, next) => {
+    const contentLength = parseInt(req.headers['content-length'], 10);
+    
+    if (isNaN(contentLength)) {
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)